tests/integration_tests/http_api/util/test_case.py

```
@@ -120,6 +120,26 @@ def create_changefeed(sink_uri):
+            "sink": {
+                "kafka_config": {
+                    "glue_schema_registry_config": {
+                        "region": "us-west-1",
+                        "registry_name": "ticdc-test"
+                    }
+                }
+            }
+        }
+    })
+    headers = {"Content-Type": "application/json"}
+    resp = rq.post(url, data=data, headers=headers)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)